src/main/scala/Main.scala

```diff
@@ -43,7 +43,12 @@ object Main {
         opt[String]('o', "outfile")
           .required()
           .action((f, c) => c.copy(output = Some(f)))
-          .text("name of the output artifact."))
+          .text("Name of the output artifact."),
+        opt[String]('i', "include")
+          .optional()
+          .unbounded()
+          .action((i, c) => c.copy(includes = i :: c.includes))
+          .text("Include location for header for CXX compilation."))
```

Comment:
Seems fine, but if you'll permit a slippery slope argument: it wouldn't do to duplicate all the `gcc` CLI flags as `fuse` flags. A lighter-weight but less convenient option would have a flag that means "pass this to the underlying compiler." Like `-x "-I path/to/header"`.

Reply:
Yeah, `-i` feels hacky. `-x` makes more sense. There is no good reason to choose one over the other.

Reply:
Fixed.